app/javascript/actionsPanel/actionsPanel.js

```diff
@@ -109,7 +109,73 @@ function updateExperienceLevel(currentUserId, articleId, rating, group) {
   );
 }
 
+function toggleSubmitContainer() {
+  document
+    .getElementById('adjustment-reason-container')
+    .classList.toggle('hidden');
+}
+
+function adjustTag(tagBtn) {
+  const reasonForAdjustment = document.getElementById('tag-adjustment-reason')
+    .value;
+  const body = {
+    tag_adjustment: {
+      // TODO: change to tag ID
+      tag_name: tagBtn.dataset.tagName,
+      article_id: tagBtn.dataset.articleId,
+      adjustment_type:
+        tagBtn.dataset.adjustmentType === 'subtract' ? 'removal' : 'addition',
+      reason_for_adjustment: reasonForAdjustment,
+    },
+  };
+
+  request('/tag_adjustments', {
+    method: 'POST',
+    body: JSON.stringify(body),
+  })
+    .then((response) => response.json())
+    .then((json) => {
+      if (json.status === 'Success') {
+        const result = json.result === 'addition' ? 'added' : 'removed';
+        tagBtn.remove();
```

Comment:
At this point, I understand that the tag in the panel is removed; think we should also remove the tag on the article itself asynchronously? 🤔

Reply:
That makes sense 👍 good point!